physics/smoke_dust/rrfs_smoke_wrapper.F90

```diff
@@ -1,18 +1,19 @@
-!>\file rrfs_smoke_wrapper.F90
+!>\file rrfs_smoke_wrapper.F90hwp_method
```

Comment:
I'm guessing that adding `hwp_method` is a mistake in this Doxygen comment?

Reply:
@grantfirl Yes, it is a mistake. Fixed. Thanks.